package.json

```diff
@@ -82,7 +82,7 @@
   },
   "peerDependencies": {
     "@babel/runtime": "^7.12.5",
-    "nebenan-ui-kit": "^5.3.0",
+    "nebenan-ui-kit": "^5.4.0",
```

Comment:
same thing here

https://github.com/goodhood-eu/nebenan-form/pull/58#discussion_r795678390

as long as we're not using new ui-kit features -> no need to change peer deps -> no breaking release